kolibri/plugins/learn/assets/src/views/SearchBox.vue

```diff
@@ -291,6 +299,17 @@
         message: 'Apps',
         context: 'Learners can filter their searches for resources by type. In this case, apps.',
       },
+      /* eslint-disable kolibri/vue-no-unused-translations */
+
+      searchPlaceholder: {
+        message: 'Search',
+        context: 'Placeholder text within the search field.',
+      },
+      findSomethingToLearn: {
```

Comment:
You have this [in another component](https://github.com/learningequality/kolibri/pull/8424/files#diff-52056ecd05d47baa126bbbd0575c256136970f699cbfea37febf02ce2f860479R217), might as well make it a core string 🙂